src/Analysis/Ast/Impl/Types/PythonType.cs

```diff
@@ -159,16 +159,14 @@ public virtual IMember Call(IPythonInstance instance, string memberName, IArgume
             }
         }
 
-        internal void MakeReadOnly() => _readonly = true;
+        internal void MakeReadOnly() {
+            lock (_lock) {
+                _readonly = true;
```

Comment:
Should `AddMember` notify somehow about `PythonType` being sealed?

Reply:
This can probably be removed. It was protection from builtins and some other types being modified during stub merge. With the reversal of stub merge direction this probably no longer needed. https://github.com/microsoft/python-language-server/issues/1503